tests/_support/Step/Acceptance/Administrator/Content.php

```diff
@@ -63,12 +63,14 @@ public function iCreateNewContent($title, $content)
 	 * Method to save an article
 	 *
 	 * @When I save an article
+	 * @When I save the category
+	 * @When I save the setting
```

Comment:
I reverted code for now, but question is open here to do something with this. :smile: